PartTwo/PartTwo/CustomView.m

```diff
@@ -0,0 +1,42 @@
+//
+//  CustomView.m
+//  PartTwo
+//
+//  Created by Daniel Distant on 9/18/15.
+//  Copyright © 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "CustomView.h"
+
+@implementation CustomView
+
+-(void)socialMediaViewDidTapLikeButton:(CustomView *)view {
+    [self.delegate socialMediaViewDidTapLikeButton:view];
+}
+
+-(void)socialMediaViewDidTapCommentButton:(CustomView *)view {
+    [self.delegate socialMediaViewDidTapCommentButton:view];
+}
+
+-(void)socialMediaViewDidTapShareButton:(CustomView *)view {
+    [self.delegate socialMediaViewDidTapShareButton:view];
+}
+
+- (IBAction)likeButtonTapped:(id)sender {
+    CustomView *view = [[CustomView alloc] init];
```

Comment:
I'm not sure what you're trying to do by creating a new instance of `CustomView`. You should not do that and you should just pass `self` as the parameter in `socialMediaViewDidTapLikeButton`.